ons_alpha/bulletins/models.py

```diff
@@ -61,22 +57,25 @@ class BulletinPage(BasePage):
         ]
     )
 
-    def get_context(self, request, *args, **kwargs):
```

Comment:
please don't remove functional code, unless there is an explicit need. And if so, do add a comment on the PR as to why it is removed.

This and the toc property are important and come from #14